presto/presto.go

```diff
@@ -74,6 +71,9 @@ import (
 	"sync"
 	"time"
 	"unicode"
+	"gopkg.in/jcmturner/gokrb5.v6/client"
```

Comment:
nit: need an empty line between them otherwise gofmt will sort again

https://github.com/golang/go/wiki/CodeReviewComments#imports

Reply:
Cool. Thanks for reviewing this. I will use goimports in GoLand